core/trino-main/src/main/java/io/trino/sql/planner/planprinter/TextRenderer.java

```diff
@@ -145,6 +151,19 @@ private String printStats(PlanRepresentation plan, NodeRepresentation node)
         return output.toString();
     }
 
+    private void printMetrics(StringBuilder output, String type, Metrics metrics)
+    {
+        if (!verbose) {
+            return;
+        }
+        Map<String, Metric> sortedMap = new TreeMap(metrics.getMetrics());
+        sortedMap.forEach((name, metric) -> {
```

Comment:
maybe format should be:
```
  connector metrics:
    'x' = yyy
    'y' = zzz
  metrics:
    'z' = vvv
```
instead of repeating `connector metrics` and `metric` in each line

Reply:
Sounds good, fixed.